docs/workflows/accessing-payload-data.md

```diff
@@ -0,0 +1,330 @@
+*   `cos` (computes the cosine of a number)
+*   `tan` (computes the tangent of a number)
+*   `asin` (computes the arcsine of a number)
+*   `acos` (computes the arccosine of a number)
+*   `atan` (computes the arctangent of a number)
+*   `sinh` (computes the hyperbolic sine of a number)
+*   `cosh` (computes the hyperbolic cosine of a number)
+*   `tanh` (computes the hyperbolic tangent of a number)
+*   `asinh` (computes the hyperbolic arcsine of a number)
+*   `acosh` (computes the hyperbolic arccosine of a number)
+*   `atanh` (computes the hyperbolic arctangent of a number)
+*   `atan2` (computes the arctangent of the two variables y and x)
+*   `sqrt` (computes the square root of a number)
+*   `log` (computes the natural logarithm of a number)
+*   `log10` (computes the base 10 logarithm of a number)
+*   `abs` (computes the absolute value of a number)
+*   `ceil` (computes the smallest integer greater than or equal to a number)
+*   `floor` (computes the largest integer less than or equal to a number)
+*   `round` (rounds a number to the nearest integer)
+*   `trunc` (truncates any fractional digits of a number)
+*   `exp` (computes e raised to the power of the given number)
+*   `pow` (raises a number to the power of a second number)
+*   `length` (computes the length of an array, object, or string)
+
+### Examples
+
+Given our example object, let's see how these expressions evaluate.
+
+```javascript
+{{data.aNumberProp}} // true
+{{data.anUndefinedProp}} // false
+{{data.aBooleanProp}} // true
+!{{data.aBooleanProp}} // false
+
+{{data.aNumberProp}} === 'hello' // false
+{{data.aNumberProp}} > 20 // true
+
+{{data}} && {{data.aBooleanProp}} // true
+{{data.aBooleanProp}} || {{data.aFalsyProp}} // true
+{{data.aBooleanProp}} && {{data.aFalsyProp}} // false
+
+{{data.aNumberProp}} > {{data.anObject.aNestedArray.[1]}} // true
+{{data.aNumberProp}} > {{data.aStringProp}} // false
+{{data.aNumberProp}} < {{data.aStringProp}} // false
+
+{{data.aFalsyProp}} == false // true
+{{data.aFalsyProp}} === false // false
+
+{{format data.aJavascriptDateObject 'MMMM'}} === 'September' // true
+{{lower data.aStringProp}} === 'Hello world' // false
+
+{{data.anObject.aNestedArray.[1]}} / 5 <= PI // false
+pow({{data.aNumberProp}}, {{data.aFalsyProp}}) === 1 // true
+```
+
+## JSON Templates
+
+In a few places within the Losant platform – such as the [Input Controls block](/dashboards/input-controls/) and [Losant API node](/workflows/data/losant-api/), it is necessary to construct JSON objects from other data. In these instances we use **JSON templates**.
+
+JSON templates can take any of the following formats (or a combination thereof):
+
+*   Valid, static JSON. e.g. `{"foo": "bar"}`, `[1,2,3]` or `null`
+*   JSON containing block helpers around keys, values or both. e.g. `{ {{#if foo}}"bar": "bat"{{else}}"baz": "bub"{{/if}} }`
+*   JSON containing string helpers for keys or values. e.g. `{"month": {{format date 'MMMM'}} }`
```

Comment:
this would not be valid, because that format guy returns a string, so it would need to be:

{"month": "{{format date 'MMMM'}}" }